swift_browser_ui_frontend/src/components/CopyFolderModal.vue

```diff
@@ -15,23 +15,18 @@
             {{ $t("message.replicate.destinationExists") }}
           </p>
         </c-alert>
-        <b-field
+        <c-text-field
+          id="new-copy-folderName"
+          v-csc-model="folderName"
           custom-class="has-text-dark"
           :label="$t('message.replicate.name_newFolder')"
-          label-for="new-copy-folderName"
-        >
-          <b-input
-            id="new-copy-folderName"
-            v-model="folderName"
-            name="foldername"
-            custom-class="has-text-dark"
-            :loading="loadingFoldername"
-          />
-        </b-field>
+          name="foldername"
+          :loading="loadingFoldername"
+        />
         <label
```

Comment:
wondering if we should keep the label, i see that CSC design system is not using labels to text fields at all, wondering if we should scrape this in favour of `label` attribute inside `c-text-field` ?

Reply:
I've been wondering that as well. They seem to use the same concept from Google's material design.

cc @hannyle, @ainoc

Reply:
Not sure I follow. 
`c-text-field` has `label` attribute. Which `label` are you referring to?

Reply:
Ahh I understand what you mean after reading the Description again.

Reply:
I personally think the `label` together with `hint` in `c-text-field` is clear enough for user.